dask/dataframe/tests/test_categorical.py

```diff
@@ -266,6 +266,30 @@ def test_categorical_set_index(shuffle):
         assert list(sorted(d2.index.compute())) == ["b", "b", "c"]
 
 
+@pytest.mark.parametrize(
+    "ncategories", list(range(1, 6)), ids=lambda x: f"{x} categories"
+)
+@pytest.mark.parametrize(
+    "npartitions", list(range(1, 6)), ids=lambda x: f"{x} partitions"
+)
+def test_categorical_set_index_npartitions_vs_ncategories(npartitions, ncategories):
+    """https://github.com/dask/dask/issues/5343"""
+    rows_per_category = 10
+    n_rows = ncategories * rows_per_category
+
+    categories = [f"CAT{i}" for i in range(ncategories)]
```

Comment:
f-strings require Python 3.6 or above, right?  If so, this will probably fail in CI.  We support Python 3.5.

Reply:
Oh right, I forgot! Fixed!